lisa/sut_orchestrator/azure/common.py

```diff
@@ -400,6 +400,30 @@ def get_data_disk_size(
         raise LisaException("No data disk feature present.")
 
 
+@dataclass_json()
+@dataclass
+class DataDiskSchema:
+    caching_type: str = field(
+        default=constants.DATADISK_CACHING_TYPE_NONE,
+        metadata=schema.metadata(
+            validate=validate.OneOf(
+                [
+                    constants.DATADISK_CACHING_TYPE_NONE,
+                    constants.DATADISK_CACHING_TYPE_READONLY,
+                    constants.DATADISK_CACHING_TYPE_READYWRITE,
+                ]
+            )
+        ),
+    )
+    size: int = 32
+    sku: str = field(
+        default=DiskType.DISK_STANDARD_HDD,
+        metadata=schema.metadata(validate=validate.OneOf(DiskType.get_disk_types())),
+    )
+    lun: int = 0
+    create_option: str = "FromImage"
```

Comment:
Will this value ever change?

Reply:
this one won't change

Reply:
Add OneOf constraint to make it's valid.

Reply:
fixed